pkg/tokens/fetcher/remote_list_of_token_lists_schema.go

```diff
@@ -0,0 +1,71 @@
+package fetcher
+
+const (
+	// #nosec G101
+	ListOfTokenListsSchema = `{
```

Comment:
probably cleaner to leave this in a `.schema.json` and load it using https://pkg.go.dev/embed